src/less/variables-payex.less

```diff
@@ -181,4 +186,9 @@ body {
     --btn-disabled-text: var(--text-disabled);
     --btn-bg-disabled: rgba(212, 196, 188, 0.3);
     --btn-active-color: var(--payex-light-blue);
+
+    //INPUT VALUES
+    --input-border-color: var(--brand-secondary-hover);
+    --input-border-focus-color: var(--brand-secondary);
+    --input-border-color-success: var(--payex-light-blue);
```

Comment:
same comment than in "src/less/variables-swedbankpay.less" ( https://github.com/SwedbankPay/design.swedbankpay.com/pull/852/files#r1155830555 ) , rename to avoid confusion